src/autocluster_k8s.erl

```diff
@@ -67,22 +67,34 @@ make_request() ->
       [{"Authorization", ["Bearer ", Token1]}],
       [{ssl, [{cacertfile, autocluster_config:get(k8s_cert_path)}]}]).
 
-%% @spec node_name(k8s_endpoint) -> list()
+%% @spec node_name(k8s_endpoint) -> list()  
 %% @doc Return a full rabbit node name, appending hostname suffix
 %% @end
 %%
 node_name(Address) ->
   autocluster_util:node_name(
     autocluster_util:as_string(Address) ++ autocluster_config:get(k8s_hostname_suffix)).
 
+
+%% @spec maybe_ready_address(k8s_subsets()) -> list()
+%% @doc Return a list of ready nodes
+%% SubSet can contain also "notReadyAddresses"  
+%% @end
+%%
+maybe_ready_address(Subset) ->
+  case proplists:get_value(<<"addresses">>, Subset) of 
+      undefined -> autocluster_log:info("No nodes ready yet!"), [];
```

Comment:
Is this a typo or we return an empty list instead of an address? Or did you mean

``` erlang
autocluster_log:info("No nodes ready yet!", [])
```